core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -87,10 +87,16 @@
   public static final String MYSQL_SOCKET_FILE_FORMAT = "/cloudsql/%s";
   public static final String POSTGRES_SOCKET_FILE_FORMAT = "/cloudsql/%s/.s.PGSQL.5432";
 
+  /**
+   * Property used to set the ApplicationName for the underlying SQLAdmin client.
+   *
+   * @deprecated Replaced by {@link #getApplicationName} and {@link #setApplicationName}.
```

Comment:
```suggestion
   * @deprecated Use {@link #setApplicationName(String)} to set the application name programmatically.
```

Most users want to set it. And it's good to use the Javadoc name with parameter types when you can.